### PR TITLE
small styles clean up

### PR DESCRIPTION
**Pull request by IAfanasovMob**
- moved absolute values to _sizes.scss
- removed unnecessary CSS classes
- added responsiveness to form input fields to match form buttons
- removed styles duplication

### CODE REVIEW

File: src/components/FormField.vue [hunk 1]
```diff
@@ -75,24 +75,24 @@ export default {
...
-    line-height: 2.6rem;
   }
 
   label {
     position: absolute;
     top: 0;
-    left: 0;
-    padding: 14px; // 4px for the border
+    left: $size-input-label-pos-left;
+    padding: $size-input-label-padding;
```
Comment (mmeester) [inline src/components/FormField.vue]: not really sure why you want to make these variables. 

This should be the only place where this is declared, if it isn't then move it to a variable

File: src/components/FormField.vue [hunk 2]
```diff
@@ -75,24 +75,24 @@ export default {
...
     overflow: hidden;
-    font-size: 1.4rem;
+    font-size: $size-input-font;
+    line-height: $size-content-font-xl + rem;
```
Comment (mmeester) [inline src/components/FormField.vue]: line-height should not be a rem, always make it a factor if the font-size is 16rem and Figma tells us that the line-height should be 24rem; the line-height is 1.5;

File: src/components/FormField.vue [hunk 3]
```diff
@@ -101,22 +101,13 @@ export default {
   }
 
   &.focused label {
-    top: -40px;
+    top: $size-input-label-focused-vertical-offset;
```
Comment (mmeester) [inline src/components/FormField.vue]: as mentioned before, I'm not seeing any added value in this.

File: src/design/_mixins.scss [hunk 4]
```diff
@@ -17,11 +16,26 @@
...
   display: inline-block;
+  padding: $size-input-padding;
+  line-height: $size-content-font-xl + rem;
   color: black;
   background-color: white;
-  border-radius: 6px;
-  @include font-size(2.6);
+  border: $size-input-border solid transparent;
+  border-color: white;
+  border-radius: $size-input-border-radius;
```
Comment (mmeester) [inline src/design/_mixins.scss]: Here I do see added value to put this in a variable 👍

File: src/components/FormField.vue (hunk 1 above)
Comment (IAfanasovMob) [reply src/components/FormField.vue]: agreed. variables removed

File: src/components/FormField.vue (hunk 2 above)
Comment (IAfanasovMob) [reply src/components/FormField.vue]: totally agree. missed that

File: src/components/FormField.vue (hunk 3 above)
Comment (IAfanasovMob) [reply src/components/FormField.vue]: removed

**Pull request merged**